azure-mgmt-containerservice/azure/mgmt/containerservice/models/orchestrator_version_profile_list_result.py

```diff
@@ -40,10 +40,11 @@ class OrchestratorVersionProfileListResult(Model):
         'id': {'key': 'id', 'type': 'str'},
         'name': {'key': 'name', 'type': 'str'},
         'type': {'key': 'type', 'type': 'str'},
-        'orchestrators': {'key': 'properties.orchestrators', 'type': 'OrchestratorVersionProfile'},
+        'orchestrators': {'key': 'properties.orchestrators', 'type': '[OrchestratorVersionProfile]'},
```

Comment:
@mboersma I guess it's what you want?

Reply:
@lmazuel yes, that looks like the important change that I've been testing with locally. 👍